File: Modul-2/PYTHON/PRAK205_2410817210001_MuhammadFadillah.py
import math

a = int(input("Masukan nilai A : "))
b = int(input("Masukan nilai B : "))

c = math.sqrt(b**2 - a**2)
keliling = a + b + c
luas = 0.5 * a * c

print(f"Alas = {c:.0f} cm")
print(f"Tinggi = {a:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: Modul-2/C/PRAK205_2410817210001_MuhammadFadillah.c
#include <stdio.h>
#include <math.h>

int main() {
    int a, b;

    printf("Masukan nilai A : ");
    scanf("%d", &a);
    printf("Masukan nilai B : ");
    scanf("%d", &b);

    int c = sqrt(b * b - a * a);
    int keliling = a + b + c;
    int luas = 0.5 * a * c;

    printf("Alas = %d cm\n", c);
    printf("Tinggi = %d cm\n", a);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm^2\n", luas);

    return 0;
}